### PR TITLE
Update release Rvt20 and readme

### PR DESCRIPTION
**Pull request by alvpickmans**
Me again. Just wanted to create a new release for Revit2020 so it is easier to access and install.
Not sure if the release I created on my branch will transfer, but probably not. @daren-thomas if you have the time, would be good if you could create a new release and set the correct path on the readme file.

Also, out of vanity, added myself to the contributors sections on the Readme (if this is going too far just let me know!) 😅

### CODE REVIEW

**Comment by daren-thomas** [comment]
@alvpickmans, I welcome your contribution :)

BTW: I'm trying to compile the "new" solution and failing - I think I'm missing some knowledge here - my .NET is super-rusty as I literally spent the last four years in a Python-only world... Maybe you could help with some pointers?

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ writing your Revit Addins - use this in combination with the [RevitLookup](https
 
 ## Installation
 
+- [Installer for Autodesk Revit 2020]((https://github.com/architecture-building-systems/revitpythonshell/releases/download/2020.01.19/2020.01.19_Setup_RevitPythonShell_2020.exe)
```
Comment (daren-thomas) [inline README.md]: brilliant! The release is not there yet - still has to be built, but I guess I should do that soon!

File: README.md [hunk 2]
```diff
@@ -87,6 +88,7 @@ This project is licensed under the terms of the [MIT License](http://opensource.
   * [Ehsan Iran-Nejad (@eirannejad)](https://github.com/eirannejad) countless improvements, the awesome [pyRevit](https://github.com/eirannejad/pyRevit) tool and a special thanks for helping maintain RPS! 
   * [@DanRumery](https://github.com/danrumery) improved autocompletion with PR #59
   * [Petr Mitev (@mitevpi)](https://github.com/mitevpi) ported to Revit 2019 with RP #86
+  * [Alvaro Ortega Pickmans (@alvpickmans)](https://github.com/alvpickmans) refactor to sdk csproject and release for Revit 2020 PR #101
```
Comment (daren-thomas) [inline README.md]: so much yes.

File: README.md (hunk 1 above)
Comment (alvpickmans) [reply README.md]: there should be an exe installer on the `Output` folder, I guess you could use that one :)

**Comment by alvpickmans** [comment]
@daren-thomas I didn't have to do much else, just have the latest VisualStudio 2019 installed (think is version 16.4.2?) and it compiled properly. Let me know if you still have issues and we can have a call maybe :)

**Pull request merged**